assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -604,3 +604,10 @@ che.workspace.devfile.default_editor=eclipse/che-theia/next
 # eclipse/che-theia-exec-plugin/0.0.1,eclipse/che-theia-terminal-plugin/0.0.1,https://cdn.pluginregistry.com/vi-mode/meta.yaml
 # If the plugin is a URL, the plugin's meta.yaml is retrieved from that URL.
 che.workspace.devfile.default_editor.plugins=eclipse/che-machine-exec-plugin/nightly
+
+# Plugin is added in case async storage feature will be enabled in workspace config
+# and supported by environment
+che.workspace.devfile.async.storage.plugin=eclipse/che-async-pv-plugin/nightly
+
+# Docker image for the Che async storage
+che.infra.kubernetes.async.storage.image=vparfonov/che-data-sync-pod:latest
```

Comment:
would be nice to set up a quay repo for https://github.com/che-incubator/workspace-data-sync